sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Grammar - <code>Inference recommender job provides</code> -&gt; <code>{The|An} inference recommender job provides</code></p><p> </p><p><code><s>We believe </s>the information provided will help you</code> -&gt; <code>The information provided will help you</code></p><p> </p><p>Helpful examples of price versus latency motivations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Fixed.